src/DataFixtures/MentoringFixtures.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\DataFixtures;
+
+use App\Entity\Mentoring;
+use DateTime;
+use Doctrine\Bundle\FixturesBundle\Fixture;
+use Doctrine\Persistence\ObjectManager;
+use Doctrine\Common\DataFixtures\DependentFixtureInterface;
+
+class MentoringFixtures extends Fixture implements DependentFixtureInterface
+{
+
+    public function load(ObjectManager $manager)
+    {
+        /**
+         * 2 mentoring relationships, with different starting dates
+         * 8 students and 8 mentors available with same professionnal sector
+         */
+
+        //mentoring relationship 1
+        $startingDate = new DateTime();
```

Comment:
I _think_ you can add the full date when you instantiate (??) a new DateTime.
It will save two lines of code.